src/Rootfs.jl

```diff
@@ -400,10 +400,15 @@ represented here, it's probably because that platform is still considered "in be
 Platforms can be excluded from the list by specifying an array of platforms to `exclude`.
 i.e. `supported_platforms(exclude=[Windows(:i686),Windows(:x86_64)])`
 """
-function supported_platforms(;exclude::Vector=[])
-    platforms = [
-        #Standard Platforms
-
+function supported_platforms(;exclude::Union{Vector{<:Platform},Function}=[])
+    exclude_platforms!(exclude::Function, platforms) = filter(!exclude, platforms)
+    function exclude_platforms!(exclude::Vector{<:Platform}, platforms)
+    	for e in exclude
```

Comment:
Intendation is messed up, should be 4 space indents, whereas this is 4 spaces and a tab. Same for the lines below.